### DISCUSSION TITLE
Is there anyway to use OMR-bypass, but failover if that route isn't available?

### DISCUSSION THREAD

**Discussion by bdruth** [Ysurac/openmptcprouter#3015] [Q&A] [answered]
Basically the title, sometimes the T-Mobile 5G Home Internet goes down and the modem has to be cycled ... when that happens, any OMR-bypass routes that are tied to that WAN just won't work. I can disable the OMR-bypass service, and then everything goes to the VPS, but ideally there'd be a way to have that just failover to another available WAN route.

**Comment by Ysurac** [comment, marked as answer]
If you want a failover then you need to not set an interface in omr-bypass. This will use master connection by default, if master is down this will failover to another interface.

**Reply by bdruth** [reply]
Oh, slick - I didn't know that the interface was optional and would use the master. Very cool, thank you!
